generic_petg.xml.fdm_material

```diff
@@ -64,6 +64,28 @@ Generic PETG profile. The data in this file may not be correct for your specific
             </hotend>
         </machine>
 
+        <machine>
+            <machine_identifier manufacturer="Ultimaker B.V." product="Ultimaker 3"/>
+            <machine_identifier manufacturer="Ultimaker B.V." product="Ultimaker 3 Extended"/>
+            <setting key="heated bed temperature">85</setting>
+            <hotend id="BB 0.4" />
+            <hotend id="BB 0.8" />
+            <hotend id="AA 0.25">
+                <setting key="hardware compatible">yes</setting>
```

Comment:
This setting is equal in all nozzles. Move to line #70 for easier reading and minimizing duplications.

Reply:
except for BB cores right?

Reply:
My mistake. BB cores do work but might ooze a bit more, don't they?